com.tracktion.Waveform13.appdata.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.tracktion.Waveform13</id>
+  <name>Waveform 13</name>
+  <icon type="stock">waveform13</icon>
+  <summary>Digital audio workstation for all music creators</summary>
+  <developer_name>Tracktion Software Corporation</developer_name>
+  <url type="homepage">https://www.tracktion.com/</url>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary=https://marketplace.tracktion.com/terms-of-use</project_license>
+
+  <description>
+    <p>
+      Waveform is a rapidly evolving application specifically designed for
+      the needs of modern music producers. Specializing in creative and
+      inspirational workflows and avoiding features not explicitly needed
+      to allow the app to remain surprisingly fun and intuitive. While
+      other apps try to appeal to broad user groups, for example film score,
+      live sound, performance - Waveform is laser focused on music production.
+    </p>
+    <p>
+      NOTE: This wrapper is not verified by, affiliated with, or supported
+      by Tracktion Software Corporation.
+    </p>
+  </description>
+
+  <categories>
+    <category>Audio</category>
+    <category>AudioVideoEditing</category>
+  </categories>
+
+  <requires>
+    <display_length compare="ge">medium</display_length>
+    <control>keyboard</control>
+    <control>pointing</control>
+  </requires>
+
+  <screenshots>
+    <screenshot type="default">
+      <caption>Modular mix screen</caption>
+      <image>https://www.tracktion.com/wp/wp-content/uploads/2017/03/waveform-screen-modular-mix.jpg</image>
+    </screenshot>
+    <screenshot type="default">
```

Comment:
there can only be one default

Reply:
there's new screenshots in the latest commit. I fixed this problem too